src/plugins/intel_cpu/src/config.h

```diff
@@ -42,7 +42,8 @@ struct Config {
     bool exclusiveAsyncRequests = false;
     bool enableDynamicBatch = false;
     SnippetsMode snippetsMode = SnippetsMode::Enable;
-    std::string dumpToDot = "";
+    std::string dumpToDot = {};
+    std::string device_id = {};
```

Comment:
Why not `int`?

Reply:
because CPU supports only `""` as device_id. All other cases lead to exception